incubator/hnc/test/e2e/e2e_test.go

```diff
@@ -9,7 +9,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
 )
 
-const namspacePrefix = "e2e-test-"
+const (
+	namspacePrefix = "e2e-test-"
+	// A 1s timeout was too short; 2s *seems* stable and also matches the Ginkgo default
+	defTimeout = 2
+	// For some operations that takes longer than 2s to finish, 5s seems to be a more stable time choice
+	longerTimeout = 5
+	// During experiments we find that there are a few cases where we need 10s timeout
+	tenSecTimeout = 10
```

Comment:
Actually I meant for something descriptive about the behavior or the reason why it takes longer for certain validation in the caller, e.g. `RunShouldContain()` with `cleanupTimeout`. I reviewed all the cases for `longerTimeout` and `tenSecTimeout`, seems they can map perfectly to `propagationTimeout = 5` and `cleanupTimeout = 10`. WDYT?

Reply:
Agree. Changed the names and comments